### PR TITLE
Task/1 pegasus math integration

### PR DESCRIPTION
**Pull request by ilia-glushchenko**

### CODE REVIEW

**Comment by ilia-glushchenko** [comment]
C++ code is just a copy of Pegasus math library with changes filenames, namespaces and include guards. The main focus of the review should not be on it.

File: demo/Main.cpp [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include <Epona/JacobiEigenvalue.hpp>
+
+int main()
+{
+    epona::JacobiEigenvalue eigenvalue(glm::dmat3(1));
+	return 0;
```
Comment (MrDmitry) [inline demo/Main.cpp]: tab -> spaces

**Review by MrDmitry — CHANGES_REQUESTED** [review]

**Review by MrDmitry — APPROVED** [review]

**Comment by ilia-glushchenko** [comment]
approve

**Comment by ilia-glushchenko** [comment]
Approve

**Comment by ilia-glushchenko** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/Godlike/Epona/pull-request/2/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=Godlike/Epona)

**Pull request merged**